pupil_src/launchables/eye.py

```diff
@@ -189,6 +192,32 @@ def get_timestamp():
         g_pool.get_timestamp = get_timestamp
         g_pool.get_now = get_time_monotonic
 
+        plugins = manager_classes + source_classes
+        g_pool.plugin_by_name = {p.__name__: p for p in plugins}
+
+        preferred_names = [
+            f"Pupil Cam3 ID{eye_id}",
+            f"Pupil Cam2 ID{eye_id}",
+            f"Pupil Cam1 ID{eye_id}",
```

Comment:
Add relevant names for PI eye cams. Maybe as an additional PR, together with changes to PI auto camera selection?

Reply:
Reference #1645